contracts-cli/index.js

```diff
@@ -0,0 +1,183 @@
+const Web3 = require('Web3')
```

Comment:
`require('web3')` with lowercase `w`, it works with MacOS because the file system is not case-sensitive but it would likely fail with Linux.